app/middlewares/authValidator.js

```diff
@@ -0,0 +1,22 @@
+const User = require('../models').user,
+  errors = require('../errors'),
+  { authValidation } = require('../constants/errorMessages'),
+  config = require('../../config'),
+  jwt = require('jwt-simple');
+
+exports.handle = (req, res, next) => {
+  if (!req.headers.token) next(errors.invalidAuthentication(authValidation.NOT_LOGGED_IN));
+  let decoded;
+  try {
+    decoded = jwt.decode(req.headers.token, config.common.session.secret);
+  } catch (e) {
+    next(errors.invalidToken(authValidation.INVALID_TOKEN));
+    return;
+  }
+  User.findOne({ where: { email: decoded.email } })
```

Comment:
add return here

Reply:
But, as this is a middleware and it's checking for user existence, if i return that result and remove the las line `next()` it never continues to the next middleware or (in this case) controller.
Let me know.